launch/Jenkinsfile

```diff
@@ -60,11 +53,6 @@ if("FAILURE".equals(currentBuild.previousBuild?.result)) {
 }
 
 
-def releaseOpenHab1Component(componentName, updateParent) {
-    def branch = env.OH1_BRANCH ?: 'master'
-    releaseOpenHabComponent(componentName, branch, env.OH1_RELEASE_VERSION, env.OH1_NEXT_VERSION, updateParent, false)
-}
-
 def releaseOpenHab2Component(componentName, updateParent) {
```

Comment:
Recently the repo was renamed from openhab2-addons to openhab-addons so the 2 can probably be removed in many places. There will also be no confusion because this PR removes releasing the OH1 from the pipeline.

Reply:
Thanks for the hint. For now, I put this PR on WIP and will try to get it fully green before we go ahead.